quantization/src/quantile.rs

```diff
@@ -22,7 +22,7 @@ pub(crate) fn find_quantile_interval<'a>(
     count: usize,
     quantile: f32,
 ) -> Option<(f32, f32)> {
-    if count < 127 {
+    if count < 127 || quantile >= 1.0 {
```

Comment:
Should we do `quantile >= 1.0 - f32::EPSILON` here instead?

Note that validation in Qdrant allows `quantile <= 1.0`.

Reply:
It should work now even with `quantile == 1.0`. That's a business logic check: if the quantile is exactly `1.0`, we don't want have exact min-max search (it's a behavior if this function returns `None`)

Reply:
Thanks for clarifying!